lib/msal-core/src/Account.ts

```diff
@@ -0,0 +1,101 @@
+ * without restriction, including without limitation the rights to use, copy, modify,
+ * merge, publish, distribute, sublicense, and/or sell copies of the Software, and to
+ * permit persons to whom the Software is furnished to do so, subject to the following
+ * conditions:
+ *
+ * The above copyright notice and this permission notice shall be
+ * included in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED 'AS IS', WITHOUT WARRANTY OF ANY KIND,
+ * EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS
+ * OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY,
+ * WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT
+ * OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+import { ClientInfo } from "./ClientInfo";
+import { IdToken } from "./IdToken";
+import { Utils } from "./Utils";
+
+/**
+ * accountIdentifier       combination of idToken.uid and idToken.utid
+ * homeAccountIdentifier   combination of clientInfo.uid and clientInfo.utid
+ * userName                idToken.preferred_username
+ * name                    idToken.name
+ * idToken                 idToken
+ * sid                     idToken.sid - session identifier
+ * environment             idtoken.issuer (the authority that issues the token)
+ */
+export class Account {
+
+    accountIdentifier: string;
+    homeAccountIdentifier: string;
+    userName: string;
+    name: string;
+    idToken: Object;
+    sid: string;
+    environment: string;
+
+    /**
+     * Creates an Account Object
+     * @praram accountIdentifier
+     * @param homeAccountIdentifier
+     * @param userName
+     * @param name
+     * @param idToken
+     * @param sid
+     * @param environment
+     */
+    constructor(accountIdentifier: string, homeAccountIdentifier: string, userName: string, name: string, idToken: Object, sid: string,  environment: string) {
+      this.accountIdentifier = accountIdentifier;
+      this.homeAccountIdentifier = homeAccountIdentifier;
+      this.userName = userName;
+      this.name = name;
+      this.idToken = idToken;
+      this.sid = sid;
+      this.environment = environment;
+    }
+
+    /**
+     * @hidden
+     * @param idToken
+     * @param clientInfo
+     */
+    static createAccount(idToken: IdToken, clientInfo: ClientInfo): Account {
+
+        // create accountIdentifier
+        // TODO: should we advocate idToken.oid+idToken.tid for all MSALs??
+        let accountIdentifier: string;
+
+        if (idToken.objectId) {
+          accountIdentifier = idToken.objectId;
+        }
+        else {
+          accountIdentifier = idToken.subject;
+        }
+
+        // create homeAccountIdentifier
+        let uid: string;
+        let utid: string;
+
+        if (!clientInfo) {
+            uid = "";
```

Comment:
we should stick to single quotes, it is convention in js community

Reply:
Sure.